packages/web-app-files/src/spaces/views/Spaces.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="oc-p-s">
+    <h2 class="" v-text="$gettext('Spaces')"></h2>
+    <div class="">
+      <span v-text="$gettext('Access all project related files on one place.')"></span>
+      <a href="#" v-text="$gettext('Learn more about spaces.')"></a>
+      <h3 v-text="$gettext('Your spaces')"></h3>
+      <hr class="uk-margin-bottom" />
+    </div>
+    <div class="spaces-list uk-container uk-container-expand uk-padding-remove">
+      <list-loader v-if="loadSpacesTask.isRunning" />
+      <template v-else>
+        <div
+          v-if="!!spaces.length"
+          class="
+            uk-grid-match uk-grid-column-small uk-grid-row-large uk-text-center uk-child-width-1-3@s
+          "
+          uk-grid
+        >
+          <a v-for="space in spaces" :key="space.id" href="#">
+            <span class="spaces-list-card oc-border uk-card uk-card-default">
+              <span class="uk-card-media-top oc-border-b">
+                <img
+                  :src="space.image ? space.image : defaultImg"
+                  :class="{ 'spaces-list-default-img': !space.image }"
+                  :alt="$gettext('Space image')"
+                />
+              </span>
+              <span class="uk-card-body">
+                <h3 class="uk-card-title">{{ space.name }}</h3>
+              </span>
+            </span>
+          </a>
+        </div>
+        <div v-else>
+          <no-content-message id="files-spaces-empty" class="files-empty" icon="space">
+            <template #message>
+              <span v-translate>You don't have any spaces currently</span>
```

Comment:
Other views have a combination of big icon  + message for the no content message. Could you do the same here as well?
Also, text could be `You don't have access to any spaces`. The point in time (`at the moment` or `currently`) is not needed in the message. Could be streamlined / simplified in other views as well but doesn't need to happen in this PR.

Reply:
Adjusted the text 👍  The icon should already be displayed due to `icon="space"` as soon as the ODS gets updated.

Reply:
ah, perfect, thanks!